ci-operator/config/openshift/kube-compare/openshift-kube-compare-main.yaml

```diff
@@ -73,7 +73,7 @@ tests:
   commands: GOFLAGS="-mod=readonly" make test-report-creator
   container:
     from: bin
-  run_if_changed: ^addon-tools/report-creator/*$
+  skip_if_only_changed: (^docs/)|((^|/)OWNERS(_ALIASES)?$)|((^|/)[A-Z]+\.md$)
```

Comment:
Maybe we should also add `/vendors` and `/hack`?

Reply:
I think `/vendors` is safe to exclude; The only time it should change is if go modules have changed, so there would always be an associated change to `go.mod` which should always trigger a build (in case a go module change would break the build and/or test).

`/hack` is trickier; If our CI ever relies on scripts in `/hack` excluding it could be dangerous.  I'm not sure if it does today or not...

Reply:
Hack has scripts for installing and running markdown and go linters.